metastats-domain/src/main/groovy/life/qbic/metastats/filter/FilterExperimentDataFileOutput.groovy

```diff
@@ -0,0 +1,15 @@
+package life.qbic.metastats.filter
+
+import life.qbic.metastats.datamodel.MetaStatsPackageEntry
+
+interface FilterExperimentDataFileOutput {
+
+
+    /**
+     * Write the created metadata into a file and download it to the user system
+     * @param entries with the metadata information of the project
+     * @return
+     */
+    def writeMetaDataToFile(List<MetaStatsPackageEntry> entries)
```

Comment:
I suggest to remove implementation details here. File is a detail.
```suggestion
    def write(List<MetaStatsPackageEntry> entries)
```

Reply:
50ecbbc